cpp/src/render/2d/heatmap/heatmap.h

```diff
@@ -1,7 +1,7 @@
 #pragma once
 
 #include "render/2d/general_2d.h"
```

Comment:
we'd better use unified include style, i.e. #include "render/2d/general_2d.h" or #include "general_2d.h", I suggest the former one